elf.lisp

```diff
@@ -834,12 +846,6 @@
    (name :initarg :name :accessor name)
    (data :initarg :data :reader data :writer set-data)))
 
-(defmethod print-object ((section section) stream)
```

Comment:
I did not intend to remove this. I initially edited the files pulled by `quicklisp`, which are probably slightly behind the `main` branch.